src/java.base/share/classes/com/sun/crypto/provider/FeedbackCipher.java

```diff
@@ -157,7 +157,7 @@ abstract int encrypt(byte[] plain, int plainOffset, int plainLen,
      */
      int encryptFinal(byte[] plain, int plainOffset, int plainLen,
                       byte[] cipher, int cipherOffset)
-         throws IllegalBlockSizeException, ShortBufferException {
```

Comment:
Why was SBE removed?

Reply:
Because IJ claims SBE is unnecessary, and indeed, this code builds fine without SBE.